axlearn/common/trainer.py

```diff
@@ -1091,8 +1093,9 @@ def _run_step(
             train_summaries=outputs["summaries"], force_runs=force_run_evals
         )
 
-        # Checkpointer policy will decide if we should save.
-        self.save_checkpoint(evaler_summaries=evaler_summaries)
+        with LoggingContext("save_checkpoint", threshold=120):
```

Comment:
Can we make it configurable? 
We should be able to patch this ad-hoc in jobs we want to debug and monitor closely. But the hard-coded 120s here makes it less flexible to apply in general jobs, hence not a good candidate for PR merge